src/Uno.UI.RemoteControl/Uno.UI.RemoteControl.netcoremobile.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
 		<TargetFrameworks>$(NetCurrentNetCoreMobile);$(NetPreviousNetCoreMobile)</TargetFrameworks>
```

Comment:
Why no `NetCoreMobilePreviousAndCurrent` - like there is with Skia & WASM?

Reply:
Good point. I think I should add this property and use it.